Feature/document address metadata (#173)
meshcloud/meshcloud-docs: docs/meshstack.configuration.md

- `allowPartnerInviteLinks` enables the use of [invite links](administration.customers.md#invite-customer-via-link)
- `approvalRequired` configures manual [customer approval](./administration.customers.md#approve-customer) through a partner

### Address Metadata

meshStack can maintain company and billing address metadata for meshCustomers and meshProjects. This is useful if your
meshStack implementation provides cloud services to 3rd parties or your cloud chargeback requires internal invoicing
between different subsidiaries that cannot be covered using e.g. cost-center numbers.

If you don't need address metadata, we recommend hiding it from end-users of meshStack by setting the `panel.environment.ui` configuration option:

```haskell
{
  hideAddress : Optional Bool
}
```

> When enabling `hideAdress`, operators should also configure the `panel.environment.ui.register.defaultAddress` option:

Reviewer: ```suggestion
When enabling `hideAddress`, operators should also configure the `panel.environment.ui.register.defaultAddress` option:
```

Author: good catch, thx!

```

The GCD Service User needs read access to the [GCD Directory API](https://developers.google.com/admin-sdk/directory/v1/get-start/getting-started).

### Customizable Descriptions

Depending on your meshStack implementation, it may be helpful for you to customize error descriptions to guide

> end-users towards resolving common problems on their own. meshStack therefore provides the following configuration

Reviewer: ```suggestion
end-users towards resolving common problems in self-service. meshStack therefore provides the following configuration
```
